### PR TITLE
♻️  Refactor Greeting Component

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
Refactor of Greeting Component Code: add useState and useEffect

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🎉 Do you have a heroku Deployment already? Then you can include the Link in your Pull Request, so we can look at the app! 🧐

File: components/greetings/greetings.tsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 
-const date = new Date();
-const currentTime = date.getHours();
-
-let greetings;
+function Greetings() {
+  const [greeting, setGreeting] = useState(null);
+  useEffect(() => {
+    const date = new Date();
+    const currentTime = date.getHours();
```
Comment (mhstrkmp) [inline components/greetings/greetings.tsx:8]:
```suggestion
    const currentTime = new Date().getHours()
```

If you only need the current time you could shorten this a little ...

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

File: components/greetings/greetings.tsx [hunk 2]
```diff
@@ -1,23 +1,24 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
```
Comment (phlgr) [inline components/greetings/greetings.tsx:1]:
```suggestion
import { useEffect, useState } from "react";
```

**Review by phlgr — CHANGES_REQUESTED** [review]
Gj 🚀

File: components/greetings/greetings.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply components/greetings/greetings.tsx:8]: Thank you, will do!

Comment (SilviaBarcellini) [reply components/greetings/greetings.tsx:8]: > LGTM! 🎉 Do you have a heroku Deployment already? Then you can include the Link in your Pull Request, so we can look at the app! 🧐

Till now I have been deploying the main branch only :) I will sort it

**Pull request merged**